mpconfigport.h: fix alloca include for NetBSD.

waggobond opened the pull request:
a minor fix to allow building on NetBSD as well as FreeBSD.

waggobond committed:
NetBSD

waggobond committed:
NetBSD

waggobond committed:
mpconfigport.h: fix alloca include for NetBSD.

waggobond committed:
mpconfigport.h: fix alloca include for NetBSD.

dpgeorge commented:
Thanks for the contribution, the fix looks good.

I know this PR is a small and simple change, but we do require a valid author email address for commits that are made to core files.  Are you able to squash and reauthor the commit with a real email address?

waggobond committed:
/mpy-cross/mpconfigport.h: include alloca.h for NetBSD.

FreeBSD had <alloca.h> but this wasn't included in build instructions
for NetBSD.

waggobond committed:
/ports/unix/mpconfigport.h: include alloca.h for NetBSD

<alloca.h> was included for FreeBSD but not NetBSD.

waggobond committed:
Merge branch 'master' of https://github.com/waggobond/micropython
